store-and-retrieve-data/networks/local-devnet-starter-kit.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Local Devnet Starter Kit
+description: Configuration details, pre-funded accounts, and MSP endpoints that help you set up your environment and start developing on the DataHaven local devnet.
+---
+
+# Local Devnet Starter Kit
+
+DataHaven is a fully EVM-compatible decentralized storage and computation platform built as a Substrate-based Layer 1 blockchain. Developers can interact with DataHaven using Ethereum RPCs, popular wallets, and libraries such as MetaMask or viem, while also accessing Substrate APIs for additional features.
+
+The following sections provide essential information for configuring your local development tools to work with the DataHaven devnet. You’ll also find details on pre-funded accounts and MSP endpoints for testing your integrations.
+
+|        Network         | Network Type | Native Asset Symbol | Native Asset Decimals |
+|:----------------------:|:------------:|:-------------------:|:---------------------:|
+| DataHaven Local Devnet | Local Devnet |         SH          |          18           |
+
+## Network Configuration
+
+To interact with the network, use the following information to configure DataHaven's local devnet within the developer tools you are using:
+
+|    Variable     |                    Value                    |
+|:---------------:|:-------------------------------------------:|
+|    Chain ID     | <pre>`{{ networks.devnet.chain_id }}`</pre> |
+| Public RPC URLs | <pre>`{{ networks.devnet.rpc_url }}`</pre>  |
+| Public WSS URLs | <pre>`{{ networks.devnet.wss_url }}`</pre>  |
+
+## Block Explorers
+
+The local devnet does not include hosted explorers. Use tools such as:
+
+- `hardhat` local chain inspection  
```

Comment:
```suggestion
- `hardhat` local chain inspection  
```

We don't typically but hardhat in backticks, usually just refer to it as Hardhat, can you pls add a link?